docusaurus/website/README.md

```diff
@@ -16,6 +16,7 @@ This website was created with [Docusaurus](https://docusaurus.io/).
 # Install dependencies
 $ yarn
 ```
+
```

Comment:
Looks good, can you remove this file?

Reply:
Hello. Yes, I think I did it.

Reply:
Oops, sorry, I thought you said this line. Removing it now.

Reply:
Sorry, did you mean that I should remove line 19, or revert the whole file?

Reply:
Sorry for the confusion! Yeah just to remove the new line that you added to `docusaurus/website/README.md` by accident... looks good now thanks!